lib/tasks/update_code_climate_projects.rake

```diff
@@ -0,0 +1,6 @@
+namespace :code_climate do
+  desc 'Updates CodeClimateProjectMetric cc_repository_id for each project'
+  task link: :environment do
```

Comment:
I think there is a way to test tasks, I used https://thoughtbot.com/blog/test-rake-tasks-like-a-boss in one project and IMO it helps to find unused code and possible bugs, even if the test just expects that `Processors::CodeClimateUpdater` receives `call`. Certainly, not a must, just a comment 🚀

Reply:
Nice article. I'll take it into account.